pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -13,6 +13,15 @@ option go_package = "github.com/buildbarn/bb-remote-execution/pkg/proto/configur
 
 message ApplicationConfiguration {
   // Configuration for blob storage.
+  //
+  // Note: some configurations have implmentation nounces that lead to a differences in runtime behavior
```

Comment:
I don't think it is right to document this here, as this is in no way specific to the blobstore API. This applies to any form of gRPC traffic. I also wouldn't describe it as an implementation nuance; it's inherent to how gRPC metadata forwarding works.

Wouldn't it make more sense to extend the description of `forward_metadata` to mention something like this?

> Keep in mind that this option can only be used to forward metadata from incoming calls on gRPC servers to outgoing calls on gRPC clients. This means that this option cannot be used to let bb-scheduler forward client provided credentials to workers.

Also note that the .proto files currently use a column limit of 80-ish characters.

Reply:
Makes sense, let me just copy verbatim your comment and send a PR to bb-storage.